src/prototype_creep_harvest.js

```diff
@@ -49,8 +49,7 @@ Creep.prototype.spawnCarry = function() {
   }
 
   var foundKey;
-  let sizes = Object.keys(config.carry.sizes).sort(function(a, b) { return parseInt(a, 10) - parseInt(b, 10); });
-  for (let key of sizes) {
+  for (let key in config.carry.sizes) {
```

Comment:
Because we have if in for-in loop. codeclimate suposed that if we have if, then we validate key

The error might be if in some moment in future, someone decide to modify `Object.prototype` for example to add some usefull method. In that case, this "usefull" method may appear in all for-in loops as key, so if someone use for-in loop, it's better to check hasOwnProperty, just in case)